cinch/roles/jenkins_master/tasks/https.yml

```diff
@@ -15,34 +15,20 @@
     mode: 0755
   with_items:
     - in: "{{ jenkins_ssl_cert }}"
-      out: jenkins.crt
+      out: "jenkins.crt"
     - in: "{{ jenkins_ssl_key }}"
-      out: jenkins.key
+      out: "jenkins.key"
```

Comment:
Strings should always be quoted. It helps preventing errors when vars contain weird characters and stuff.